scala/.gitignore

```diff
@@ -0,0 +1,43 @@
+# Operating System Files
+
+# *.DS_Store
+Thumbs.db
+
+# Build Files
+
+bin
+target
+build/
+.gradle
+
+# Eclipse Project Files
+
+.classpath
+.project
+.settings
+
+# IntelliJ IDEA Files
+
+*.iml
+*.ipr
+*.iws
+*.idea
+
+# Spring Bootstrap artifacts
+
+dependency-reduced-pom.xml
+README.html
+
+# Sublime files
+
+*.sublime-workspace
+
+# Test data files #
+
+java/data
+
+# Compiled libs #
+
+java/*.dylib
+java/*.so
+java/*dll
```

Comment:
It's the same .gitignore; i think it makes sense to move it into java directory, as a subproject of already existence structure.